includes/class-wc-gateway-ppec-client.php

```diff
@@ -460,7 +460,7 @@ protected function _get_details_from_cart() {
 		$discounts     = WC()->cart->get_cart_discount_total();
 
 		$details = array(
-			'total_item_amount' => round( WC()->cart->cart_contents_total, $decimals ) + $discounts,
+			'total_item_amount' => round( WC()->cart->cart_contents_total + $discounts, $decimals ),
```

Comment:
Why do we need to add rounding in `_get_details_from_cart()` and `_get_details_from_order`? I'm asking because the `total_item_amount` gets rounded correctly in `get_details` which is called from both of these functions.